content/en/blog/2019/alb-ingress-gateway-iks/index.md

```diff
@@ -0,0 +1,394 @@
+on [IBM Cloud Kubernetes Service (IKS)](https://www.ibm.com/cloud/kubernetes-service/) to direct traffic to the Istio
+ingress gateway, while securing the traffic between them using {{< gloss  >}}mutual TLS authentication{{< /gloss >}}.
+
+When you use IKS without Istio, you may control your ingress traffic using the provided
+ALB. This ingress-traffic routing is configured using a Kubernetes
+[Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resource with
+[ALB-specific annotations](https://cloud.ibm.com/docs/containers?topic=containers-ingress_annotation). IKS provides a
+DNS domain name for each cluster. The domain name is in the following format: `<cluster_name>.<region>.containers.appdomain.cloud`, for example
+`mycluster.us-south.containers.appdomain.cloud`. In addition to registering a DNS domain, IKS provides a TLS certificate
+that matches the domain, and a private key for the certificate. IKS stores the certificates and the private key in a
+[Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/).
+
+When you start using Istio in your IKS cluster, the recommended way is to use the
+[Istio Ingress Gateway](/docs/tasks/traffic-management/ingress/ingress-control/) instead of using the
+[Kubernetes Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/). One of the main reasons to use
+Istio ingress gateway is the fact the ALB provided by IKS will not be able to
+communicate directly with the services inside the mesh when you enable Istio mutual TLS. During your transition to
+having only Istio ingress gateway as your main entry point, you can continue to use the traditional Ingress for
+non-Istio services while using the Istio ingress gateway for services that are part of the mesh.
+
+IKS provides a convenient way for clients to access Istio ingress gateway by letting you
+[register a DNS domain](https://cloud.ibm.com/docs/containers?topic=containers-loadbalancer_hostname) for the gateway's
+IP with an IKS command. The domain is in the following [format](https://cloud.ibm.com/docs/containers?topic=containers-loadbalancer_hostname#loadbalancer_hostname_format): `<cluster_name>-<globally_unique_account_HASH>-0001.<region>.containers.appdomain.cloud`, for example `mycluster-a1b2cdef345678g9hi012j3kl4567890-0001.us-south.containers.appdomain.cloud`. In the same way as for the ALB domain, IKS provides a certificate and a private key,
+storing them in another Kubernetes secret.
+
+In this blog post you configure the IKS Ingress ALB to direct traffic to the services inside an Istio service mesh
+through the Istio ingress gateway, while using mutual TLS authentication between ALB and the gateway. For the mutual TLS authentication, you configure ALB and the Istio ingress gateway to use the certificates and the keys provided by IKS.
```

Comment:
```suggestion
through the Istio ingress gateway, while using mutual TLS authentication between ALB and the gateway. For the mutual TLS authentication, you will configure ALB and the Istio ingress gateway to use the certificates and keys provided by IKS for ALB and NLB host names. You will use the NLB certificate as the server certificate for Istio ingress gateway and use the ALB certificate ALB as the client certificate of ALB . You then use a
[Let's Encrypt](https://letsencrypt.org) certificate as the root for both client and server certificates.
```

Reply:
make sure to add `the` before ALB (multiple instances of this throughout this blog), so always `the ALB` or `an ALB` etc

Reply:
Fixed by https://github.com/istio/istio.io/pull/5098/commits/7adfa2fbb2fa2ce5f221c3ea05cfe12bc612c1a6.